src/pages/uxp/reference-js/Modules/FileSystem.md

```diff
@@ -101,16 +101,16 @@ const strLen = await fs.writeFile("plugin-data:/textFile.txt", "It was a dark an
 Writes data to a path synchronously, appending if desired.
 The format of the file is controlled via the encoding option, and defaults to a binary format.
 
+**Returns**: `int` - the length of contents written to the file  
```

Comment:
https://github.com/AdobeDocs/uxp/issues/86 should be `number`

Reply:
Is the return type just `number` or `Promise<number>` ?